lib/sycamore/sycamore/materialize.py

```diff
@@ -286,14 +290,18 @@ class AutoMaterialize(NodeTraverse):
        a = AutoMaterialize({"root":Path|str, fs=pyarrow.fs, name=fn, clean=bool, tobin=fn})
        ctx.rewrite_rules.append(a)
 
+       # The source_mode can be passed to AutoMaterialize, which will in turn be passed to all
+       # created materialize nodes. To use each materialized node as a source:
+       ctx.rewrite_rules.append(AutoMaterialize(source_mode=MaterializeSourceMode.USE_STORED)
```

Comment:
`source_mode=sycamore.MATERIALIZE_USE_STORED`

both work, but that form involves less importing/typing
it can't be used for internal stuff because of circular dependencies.